src/jsTest/kotlin/it/krzeminski/githubactionstyping/JsonlintTest.kt

```diff
@@ -0,0 +1,25 @@
+package it.krzeminski.githubactionstyping
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.mpp.env
+import prantlf.jsonlint.ParseOptions
+import prantlf.jsonlint.parse
+import node.buffer.BufferEncoding.Companion.utf8
+import node.fs.readFile
+
+private val schemaFile = env("schemaFile")!!
+
+class JsonlintTest : FunSpec({
```

Comment:
nit: again, the name - it's about linting the schema, right? So e.g. `SchemaLintingTest`? Otherwise it looks like you'd test `jsonlint`.